### DISCUSSION TITLE
test  Sunshine while still using gamestream

### DISCUSSION THREAD

**Discussion by ermac500** [LizardByte/community#121] [Sunshine] [answered]
I haven't tried yet because I'm still using gamestream until its termination, but I want to test sunshine meanwhile.
will this work or having both enabled will cause issues?

**Comment by yungleballz** [comment, marked as answer]
Should work if you change the ports for sunshine.
However, you can easily switch between them (might need router/host reboot, especially for remote connections).

**Comment by ermac500** [comment]
I was planning on using different ports for sunshine anyway (since it has that possibility unlike gamestream).
My work place has some of the gamestream ports blocked on LAN and wifi is kinda unstable.
thanks, I'll give it a go after the hollydays.
hope sunshine will get into a place that will have nobody missing gamestream, if it's not there already.

**Reply by yungleballz** [reply]
It's there already, atleast the nightly build. v0.16 is still slower.

**Reply by ermac500** [reply]
where do I get the nightly build from? is it the CI build?

**Reply by yungleballz** [reply]
https://github.com/LizardByte/Sunshine/releases/tag/nightly-dev

**Comment by ReenigneArcher** [comment]
You can just disable gamestream and then start sunshine. No restart required from my experience.
